backend/typescript/services/implementations/authService.ts

```diff
@@ -182,21 +184,26 @@ class AuthService implements IAuthService {
     try {
       const authId = await this.userService.getAuthIdById(user.id);
 
-      const setPasswordLink = `${process.env.CLIENT_URL}/auth/action?mode=verify-user&uid=${authId}`;
+      const setPasswordLink = `${process.env.CLIENT_URL}/auth/action?mode=verify-user&uid=${authId}&new-account=${isNewAccount}`;
```

Comment:
Is there any way to not have this parameter, and have the frontend determine whether this is a new account? This seems like a security risk. 
Also, we should probably discuss sending some random token instead of the authID to improve security, but that might be a separate PR

Reply:
resolved for now as discussed